packages/webcert/src/feature/careProvider/CareProviderModal.tsx

```diff
@@ -1,21 +1,50 @@
 import { PopUpModal, ResourceLinkType } from '@frontend/common'
 import _ from 'lodash'
-import React from 'react'
-import { useSelector } from 'react-redux'
-import { getUserResourceLinks, selectIsLoadingUserStatistics } from '../../store/user/userSelectors'
+import React, { useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { updateIsCareProviderModalOpen } from '../../store/user/userActions'
+import { getUser, selectIsLoadingUserStatistics, getIsCareProviderModalOpen, getUserResourceLinks } from '../../store/user/userSelectors'
 import { CareProviderModalContent } from './CareProviderModalContent'
 
 const CareProviderModal: React.FC = () => {
+  const dispatch = useDispatch()
+  const user = useSelector(getUser)
   const isLoadingUserStatistics = useSelector(selectIsLoadingUserStatistics, _.isEqual)
+  const isCareProviderModalOpen = useSelector(getIsCareProviderModalOpen)
```

Comment:
Is it necessary to save this variable in the state? Why can't a local variable be used?